benchmarks/benchmark/tools/locust-load-inference/locust-docker/locust-tasks/tasks.py

```diff
@@ -107,7 +107,7 @@ def lm_generate(self):
         prompt = test_data[random.randrange(0, len(test_data))]
 
         request = generate_request(model_params, prompt)
-        headers = {"User-Agent": "Benchmark Client"}
+        headers = {"User-Agent": "Benchmark Client", 'Connection':'close'}
```

Comment:
can you use double quotation marks for consistency with the other strings?

Reply:
done!